x-pack/plugins/snapshot_restore/public/app/sections/home/snapshot_list/snapshot_list.tsx

```diff
@@ -154,6 +172,7 @@ export const SnapshotList: React.FunctionComponent<RouteComponentProps<MatchPara
           repositories={repositories}
           reload={reload}
           openSnapshotDetails={openSnapshotDetails}
+          filterToRepository={filteredRepository}
```

Comment:
Minor nit: I think a prop name of  `repositoryFilter` would make more sense to me, since this is just the string that gets applied as a filter for the repository field. `filterToRepository` confused me a bit because it sounded like maybe it did more than that.